gallery/gallery/lib/pages/backdrop.dart

```diff
@@ -13,17 +13,92 @@ import 'package:gallery/constants.dart';
 import 'package:gallery/data/gallery_options.dart';
 import 'package:gallery/l10n/gallery_localizations.dart';
 import 'package:gallery/layout/adaptive.dart';
+import 'package:gallery/pages/home.dart';
+import 'package:gallery/pages/settings.dart';
+
+class AnimatedBackdrop extends StatefulWidget {
+  @override
+  _AnimatedBackdropState createState() => _AnimatedBackdropState();
+}
+
+class _AnimatedBackdropState extends State<AnimatedBackdrop>
+    with SingleTickerProviderStateMixin {
+  AnimationController backdropController;
+  ValueNotifier<bool> isSettingsOpenNotifier;
+
+  @override
+  Widget build(BuildContext context) {
+    final openSettingsAnimation = CurvedAnimation(
```

Comment:
Should these 2 animations not be initialized in `initState` instead?

Reply:
That works too, I'll do that.